tools/executors/translate/README.md

```diff
@@ -0,0 +1,33 @@
+# Translate executor
+
+## Google Translate Setup
+
+1. Follow guidelines from: <https://cloud.google.com/translate/docs/setup>
+2. [Generate keys](https://cloud.google.com/translate/docs/setup#creating_service_accounts_and_keys) and download them
+3. Place JSON at `./files/google-cloud-secret.json`.
+
+## Usage
+
+### Modify `workspace.json` file
+
+Check / change below options under "translate" object:
+
+- `path` - List of file paths of which translation needs to be generated
```

Comment:
There is no need for a path anymore. We made the i18n addon to know which files to use seamlessly.

Reply:
Until we create json for each file, it's needed.